site/en/docs/android/custom-tabs/guide-ui-customization/index.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: "layouts/doc-post.njk"
+title: Customizing the UI
+seoTitle: "Guide: customizing the Custom Tab UI"
+date: 2023-04-21
+description: How to make a Custom Tab match the look and feel of your app.
+authors:
+  - sebastianbenz
+---
+
+One advantage of Custom Tabs is that it can be seamlessly integrated into your app. In this part of the Custom Tabs guide, you will learn how to change the appearance and behavior of a Custom Tab to match your app.
```

Comment:
> is that <s>it</s> can be seamlessly

is that _they_ can be seamlessly (I think - I read it as a collective noun, rather than singular)

Reply:
fxied